### DISCUSSION TITLE
How does all of this really work exactly ?

### DISCUSSION THREAD

**Discussion by Suyashjain05** [tomer8007/widevine-l3-decryptor#45] [Q&A]
So recently i found out that the extension ain't working anymore, as i can resolve from older issues , it's gotta do something with either chrome's new update or windows. So i was just curious about how can i find new keys instead of rolling back. I'd really appreciate if you'd draft that write up you were talking about.
Thank You

**Comment by tomer8007** [comment]
It's because of the CDM update, You can look at issue #39.

**Reply by Suyashjain05** [reply]
well if i'd be bit more specific then how to find out those keys on our own ? instead of replacing the files again and again.

**Reply by tomer8007** [reply]
I don't want you to find the keys again. This was a single-round game. 
Anyway I'm currently looking into the new CDM to see how much harder it got.

**Reply by Suyashjain05** [reply]
> I don't want you to find the keys again. This was a single-round game.
> Anyway I'm currently looking into the new CDM to see how much harder it got.

alright .
